packages/hint/docs/contributor-guide/getting-started/architecture.md

```diff
@@ -1,78 +1,81 @@
 # Architecture
 
-`webhint` was designed with extensibility in mind. At a high level, this is
-the project's architecture:
-
-[![webhint's architecture](images/architecture.svg)](images/architecture.svg)
-
-The main piece is `webhint`. It's an object that extends from `EventEmitter`.
-This means that the communication between all the different parts is going to
-be done mostly via [`event`s][events]. The other pieces are:
+The core concepts in `webhint` are:
 
 * `hint`: Is a group of related tests that are run on a resource (HTML,
-  document, image, request, etc.). E.g.: Verify that the HTML document has a
-  valid language declared.
-  [Learn how to develop a `hint`][new hint].
-* `connector`: Is the way in which `webhint` obtains information about the
-  DOM, network information, resources, etc. The underlying technique (debugging
-  protocol, web driver, etc.) to access this data does not matter to the rest
-  of the system.
-  [Learn how to develop a `connector`][new connector].
-* `formatter`: Transforms the results into something useful to the user. It
-  could be as simple as printing out the results to the command line, or
-  something more complex like creating an HTML report.
-  [Learn how to developer a `formatter`][new formatter].
+  document, image, request, etc.). E.g.: Verify that the HTML document
```

Comment:
Add `configuration files` to the list.

Reply:
I wasn't sure about the configuration files as those are just JSON files in the system, ie: we can run webhint without a config file as long as we pass the resources.
I see you have other comments about `confiurations` that I think should be addressed in another PR.

Reply:
Configuration files as in `.<something>rc` files.

Reply:
Ahh, got it.

Reply:
Done